_posts/2018-11-27-tide-middleware-evolution.md

```diff
@@ -240,4 +240,4 @@ Finally, a shout out to the 19 people (!) who have already contributed to Tide:
 - Yoshua Wuyts
 - csmoe
 - ibaryshnikov
-- lixiaohui
\ No newline at end of file
+- lixiaohui
```

Comment:
I have no idea why this is considered as diff (I'm editing the post on GitHub directly).

Reply:
Probably adding a trailing newline, normally there's a symbol displayed on the line to highlight that though 😕

Reply:
Yep, checked locally and that's what it is, checking the page source the symbol is there but set to `display: hidden` for some reason.